### ISSUE TITLE
Add verification of tck.dist module

### ISSUE THREAD

**Issue by scottkurz** [WASdev/standards.jsr352.jbatch#35]
The tck.exec module accomplishes the goal of verifying that the RI passes the TestNG (runtime) portion of the TCK.    

What we don't have yet is automation testing the TCK distribution itself.  For this we need to run the TCK in the official/external way using the Ant build.xml, rather than using the failsafe plugin to execute the TestNG tests more directly.    We also should include automation for running the SigTest portion.     (True, this will be a combined test of the RI + TCK..not purely a test of the TCK distribution).

Assuming we add this, we might not need to run the tests using failsafe as well.

**Comment by scottkurz** [comment]
I'm working on this now, btw

**Comment by scottkurz** [comment]
OK, I just pushed my refactoring.   Gave a short explanation in the updated README.md of how I see the distinction between the "official" TCK zip distribution executed via Ant (executed against the RI in com.ibm.jbatch.tck.dist.exec) existing alongside a more direct execution of the TestNG tests in Maven (executed against the RI in com.ibm.jbatch.tck.exec).

Perhaps ultimately we don't need both.   But given that the official form to date has been the zip executed via Ant, it's a smaller step to continue to support both approaches for now.

Thanks @BrentDouglas again for all your help here.   

Sorry I crippled the SigTest generation some in refactoring it... maybe we can get back to that. 

Would be happy to hear your feedback.  Also, I'm thinking of release a 1.0.1-beta1 version of the TCK group's artifacts in the next couple of days to get feedback from the public ML.    

Thanks
